src/lib.rs

```diff
@@ -87,6 +87,10 @@ impl PublicKey {
 
     /// Verify a BLS signature
     pub fn verify(&self, message: &[u8], signature: &Signature) -> Result<(), ()> {
+        if self.pk.is_identity().into() {
+            return Err(());
+        }
```

Comment:
Optionally, we could also add a test, just like in https://github.com/dfinity/vetkeys/pull/141.